File: examples/extending-objects/Inheritance/C-Sharp/Employee.cs
public class Employee : Person {
  private int staffNumber;
  
  Employee(String firstName, String lastName, int age, int staffNumber) {
    base(firstName, lastName, age);
    this.staffNumber = staffNumber;
  }
  
  public String ToString() {
    return base.ToString() + ", " + this.staffNumber;
  }
}

File: examples/extending-objects/Inheritance/JavaScript (ES6)/employee.js
//Assuming the import of the person.js file is available

class Employee extends Person {

    constructor(first, last, age, staffnum) {
        super(first, last, age)
        self.staffnumber = staffnum
	}
		
    whoAmI() {
        return this.first + " " + this.last + ", " +  self.staffnumber
	}
}